data-ingestion-and-preparation/v3io-objects.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Using Iguazio's Data-Object API"
+    "# Using Paltform's Data-Object API"
```

Comment:
typo

Reply:
FIXED (also added "the" before "Platform's") — commit 6611b5c.